user-modules/gpg-minimal.nix

```diff
@@ -0,0 +1,33 @@
+{ pkgs, ... }:
+{
+  programs.git = {
```

Comment:
this file needs a better file name / better file structure. gpg-minimal does not include ssh, git and other cli setups. The content here is better suited to be put into a general shell config and is also heavily duplicating gpg.nix at the moment. Just put it into a shell.nix